Description: Remove authentik.enterprise from 'TENANT_APPS' in authentik.root.settings.py
.
Remove enterprise related source code from build.
Author: Matthew Ramirez <matthew.ramirez@chainguard.dev>
Forwarded: not-needed

diff --git a/authentik/root/settings.py b/authentik/root/settings.py
index ac511e9..ed0dd5f 100644
--- a/authentik/root/settings.py
+++ b/authentik/root/settings.py
@@ -84,7 +84,6 @@ TENANT_APPS = [
     "authentik.crypto",
     "authentik.endpoints",
     "authentik.endpoints.connectors.agent",
-    "authentik.enterprise",
     "authentik.events",
     "authentik.admin.files",
     "authentik.flows",
